hazelcast/src/main/java/com/hazelcast/client/impl/management/ManagementCenterService.java

```diff
@@ -56,6 +58,12 @@
  */
 public class ManagementCenterService {
 
+    /**
+     * Internal property for enabling MC client mode ({@link ConnectionType#MC_JAVA_CLIENT}).
+     */
+    public static final HazelcastProperty MC_CLIENT_MODE_PROP
```

Comment:
So, if I am a bad user, I can bypass the selector by setting this prop. Is this OK?

Reply:
It's not meant as a security setting (at least for now - but in future we may add dedicated permissions for MC/admin clients). So, I don't think that's a big problem.

Also, it's relatively easy to implement a unisocket client that would authorize as an MC client. But that client (just as MC or any other client) will have to authenticate if security is enabled on the cluster side.

WDYT?

Reply:
Some more thoughts on the topic.

Client filtering is not a security feature. It's main use case - blue/green deployments. And as MC orchestrates client filtering lists in such blue/green deployments scenarios, we must be confident that MC's own client doesn't get disconnected because of the applied client filter.